armv7hf/0.10/Dockerfile

```diff
@@ -2,12 +2,12 @@
 FROM resin/armv7hf-buildpack-deps:jessie
 
 ENV NODE_VERSION 0.10.36
-ENV NPM_VERSION 2.5.0
+ENV NPM_VERSION 2.7.1
 
-RUN curl -SLO "http://resin-packages.s3.amazonaws.com/node/v$NODE_VERSION/node-v$NODE_VERSION-linux-armv7hf.tar.gz" \
+RUN curl -SLO "http://resin-packages.s3.amazonaws.com/node/v0.10.36/node-v0.10.36-linux-armv7hf.tar.gz" \

```

Comment:
I did replace the node version and arch of this line in the script. Will change Node_version back to keep the template same as below lines.